dhis2/line-listing-app
# fix(deps): platform fixes for login redirect [DHIS2-15320]

Implements [DHIS2-15320](https://dhis2.atlassian.net/browse/DHIS2-15320),
and see parent issue [DHIS2-15316](https://dhis2.atlassian.net/browse/DHIS2-15316)

- [x] Custom app build tested on `debug`

**Testing instructions:**

Note that this can't be tested on Netlify, because redirects are handled differently there 😢 
* Here are some instructions: [LIBS-473](https://dhis2.atlassian.net/browse/LIBS-473?focusedCommentId=182691)
* Here's a bundle you can install on a `debug` instance:
[line-listing-pr-366-v1.zip](https://github.com/dhis2/line-listing-app/files/12765361/line-listing-pr-366-v1.zip)

**Before:**

https://github.com/dhis2/line-listing-app/assets/49666798/60357399-6b75-4bfd-92f0-c6a35767225f

**After:**

https://github.com/dhis2/line-listing-app/assets/49666798/39732750-7a94-45b2-b5d2-c099ffab1c07

[DHIS2-15320]: https://dhis2.atlassian.net/browse/DHIS2-15320?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DHIS2-15316]: https://dhis2.atlassian.net/browse/DHIS2-15316?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[LIBS-473]: https://dhis2.atlassian.net/browse/LIBS-473?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
